app/furniture/spotlight/image.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Spotlight
+  class Image < Item
+    has_one_attached :file
```

Comment:
It would be nice if this was namespaced to the space so it is easier to see what is owned by the space

Reply:
We did some research, and this is not possible to do with ActiveStorage out of the box, so we decided this is not worth it for now.